src/main/scala/com.snowplowanalytics/weather/providers/darksky/Responses.scala

```diff
@@ -0,0 +1,71 @@
+package com.snowplowanalytics.weather
+package providers.darksky
+
+import io.circe.generic.JsonCodec
+
+object Responses {
+
+  @JsonCodec final case class DarkSkyResponse(latitude: Float,
+                                              longitude: Float,
+                                              timezone: String,
+                                              currently: Option[DataPoint],
+                                              minutely: Option[DataBlock],
+                                              hourly: Option[DataBlock],
+                                              daily: Option[DataBlock],
+                                              alerts: Option[Seq[Alert]],
+                                              flags: Option[Flags])
+      extends WeatherResponse
+
+  @JsonCodec final case class DataBlock(data: Seq[DataPoint], summary: Option[String], icon: Option[String])
+
+  @JsonCodec final case class Alert(description: String,
+                                    expires: Long,
+                                    regions: Seq[String],
+                                    severity: String,
+                                    time: Long,
+                                    title: String,
+                                    uri: String)
+
+  @JsonCodec final case class Flags(`darksky-unavailable`: Option[String], sources: Seq[String], units: String)
```

Comment:
mmh should we write a custom decoder? I don't really like those kinds of variable name

Reply:
also should `units` map back to `Units`?